src/Sparrow.Server/Collections/SmallSet.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Buffers;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+
+namespace Sparrow.Server.Collections
+{
+    public sealed class WeakSmallSet<TKey, TValue> : IDisposable 
```

Comment:
What is this meant to be used for? We need a comment explaining that.

Reply:
@redknightlois ping

Reply:
Github is acting up. This is already fixed, it just didnt show as outdated.